packages/core/forms/src/components/fields/FieldInput.vue

```diff
@@ -85,6 +102,14 @@ const emit = defineEmits<{
 }>()
 
 const propsRefs = toRefs(props)
+const eyeOpen = ref(false)
+
+const displayInputType = computed(() => {
```

Comment:
Do we have a specific reason for using a separate `displayInputType` rather than directly modifying the `inputType`?

Reply:
I'd suggest we stick to `inputType` because it is the single source of truth in terms of determining the type of the input. IMO having another `displayInputType` introduces more complexity.